### DISCUSSION TITLE
Typo on ActionExecutorClient.py

### DISCUSSION THREAD

**Discussion by isacg5** [PlanSys2/ros2_planning_system#313] [General]
Hi,
I am developing a PDDL plan by using python for some nodes. I was having a problem while replanning, until I found out there is a typo in the ActionExecutorClient.py node. It is only necessary to remove one 'self.', in this way:

```cpp
            if (self._state_machine.current_state[0] == State.PRIMARY_STATE_ACTIVE and
                    msg.node_id == self.get_name()):
```
https://github.com/PlanSys2/ros2_planning_system/blob/c063ce54cf86f46594e9bda160e01dc8c40d4de5/plansys2_support_py/plansys2_support_py/ActionExecutorClient.py#L101

Hope it can save time for someone in the future :)

**Comment by fmrico** [comment]
Hi @isacg5 

Can you make a PR to fix It?

Best
